web-api/nishiki-web-api.yaml

```diff
@@ -194,6 +201,7 @@ paths:
                   type: string
                   nullable: false
                   example: bouillon
+                  required: true
```

Comment:
It's complained as a syntax error. But I'm not sure if the "nullable: false" can meet the demanded behavior.